Runtime/Scripts/RGBotLocalRuntime/RGBotAssetsManager.cs

```diff
@@ -71,6 +72,9 @@ public void RefreshAvailableBots()
                     RGDebug.LogWarning($"Bot at path `{botDirectory}` could not be loaded: {ex}");
                 }
             }
+#else
+            //TODO: can't use asset database in real runtime
```

Comment:
Assuming a ticket will be made for this follow up exploration

Reply:
it was, and was updated.. you just beat me to it :D